src/btree/bt_split.c

```diff
@@ -1073,6 +1074,7 @@ __split_internal(WT_SESSION_IMPL *session, WT_PAGE *parent, WT_PAGE *page)
 
     /* The split is complete and verified, ignore benign errors. */
     complete = WT_ERR_IGNORE;
+    F_SET_ATOMIC_16(page, WT_PAGE_INTL_PINDEX_UPDATE);
```

Comment:
This is not necessary; I can remove it. I only included it for the sake of consistency.

Reply:
I think we remove that. It confuses me why we set the flag on the page instead of parent?

Reply:
The reason I have added it is because we wish to set such a flag in any instance where the pindex was updated to a new pindex. This maintains some consistency. However, since it's confusing, I am happy to remove it.

Reply:
Updated in [78d0d5b](https://github.com/wiredtiger/wiredtiger/pull/11676/commits/78d0d5b96a4f72194768203551f5fb40d43527e2)